Build Home Page - W50-220

ridham-virufy opened the pull request:
Developed the “Homepage” web page based on the provided Figma design.

[Figma Mobile version](https://www.figma.com/design/CJVubKy4QZETjVty3Ak4eK/Website-6.0?node-id=671-5008&t=JkiBqk6C297k46Ly-1)
[Figma Desktop Version](https://www.figma.com/design/CJVubKy4QZETjVty3Ak4eK/Website-6.0?node-id=331-1815&t=wMetGCxZY361l2E6-1)

Mobile Screenshots for reference:
<img width="258" alt="Screenshot 2024-06-11 at 10 22 10 AM" src="https://github.com/virufy6/virufy6.github.io/assets/165954957/9439213a-31a5-45a1-a202-54cac9289933">
<img width="209" alt="Screenshot 2024-06-11 at 10 22 30 AM" src="https://github.com/virufy6/virufy6.github.io/assets/165954957/f4405843-413a-481e-a803-ef9f194d81f9">
<img width="209" alt="Screenshot 2024-06-11 at 10 22 36 AM" src="https://github.com/virufy6/virufy6.github.io/assets/165954957/068cc1f3-7c9e-4d5b-a595-e83e3ad68bb0">


Let me know if there are any suggestions!

I do need help figuring out why the header and footer are not looking the desired way, that is having a transparent background. It might work once Homepage and the updated header by @a-wong-8 is merged together into the release.

ridham-virufy committed:
section 1 updates

ridham-virufy committed:
Merge pull request #29 from virufy6/release

pulling from release

ridham-virufy committed:
responsivness of section1

ridham-virufy committed:
Merge branch 'Homepage-ridham' of https://github.com/virufy6/virufy6.github.io into Homepage-ridham

ridham-virufy committed:
extra div removed

ridham-virufy committed:
section2 partially complete

ridham-virufy committed:
section2 part2

ridham-virufy committed:
still working on section2

ridham-virufy committed:
still working

ridham-virufy committed:
last seccond commit

ridham-virufy committed:
mobile frame updated

ridham-virufy committed:
final touches

ridham-virufy committed:
final changes completed

a-wong-8 reviewed (commented):
good job @ridham-virufy ! visually, the only problem I see so far is the text running into the img in tablet view. I havent looked at the code yet. 

![Screenshot 2024-06-11 at 9 52 31 AM](https://github.com/virufy6/virufy6.github.io/assets/127047078/c7a38e17-3696-4982-8d5d-f13618d8ccd0)

gevuong commented:
can a max width be set for the page? That way, the content will not continue to stretch beyond a certain width. I see this occurring at 2560px (4k). I suggest using a max width of `max-w-5xl`.

For context:
<img width="1282" alt="Screenshot 2024-06-13 at 5 55 50 PM" src="https://github.com/virufy6/virufy6.github.io/assets/27314227/59a64cfa-9366-42fd-854a-f40ce99446cc">
<img width="1283" alt="Screenshot 2024-06-13 at 5 57 46 PM" src="https://github.com/virufy6/virufy6.github.io/assets/27314227/6722a0f7-366b-4779-b699-9914eb28b43e">

Other pages:
<img width="1282" alt="Screenshot 2024-06-13 at 5 57 22 PM" src="https://github.com/virufy6/virufy6.github.io/assets/27314227/9716e3af-6d30-4835-b95a-1279bb074a8e">
<img width="1286" alt="Screenshot 2024-06-13 at 5 56 37 PM" src="https://github.com/virufy6/virufy6.github.io/assets/27314227/586b4a5a-7e1a-418c-bd14-bd892c520cfb">

a-wong-8 reviewed (commented):
the code looks good to me. only thing is fix the thing I mentioned above 👍🏼

gevuong commented on src/components/organisms/section-10/Section10.js:
```diff
@@ -0,0 +1,43 @@
+    subtitle,
+    text
+}) {
+  const { } = useI18n();
```
Remove if unused, including its import

gevuong commented on src/pages/home.tsx:
```diff
@@ -1,106 +1,57 @@
+          sizeImg="width-100vw"
+          borderImg="none"
+          src={ImageSection2}
+          altImg="Imagen cabezera"
```
should this be in English?

gevuong commented on src/components/organisms/section-3/Section3.js:
```diff
@@ -1,51 +1,97 @@
+    disclaimer,
+    buttonText
+}) {
+  const { } = useI18n();
```
remove if unused, including its import

gevuong commented:
Looks good! Consider setting a max width to prevent content from stretching, and to match the other pages. Note that the hardcoded content will eventually need to be abstracted to support other languages (similar to the other updated pages). But that can be done in a future revision.

ridham-dd committed:
closed comments from the code review

ridham-virufy commented:
- Made the content aligned centered and set a max width.

![image](https://github.com/virufy6/virufy6.github.io/assets/165954957/10fea339-3dbd-437b-adc0-ad4a5721e3f5)


- Now the text is not running into the img in tablet view

<img width="699" alt="Screenshot 2024-06-18 at 11 39 13 AM" src="https://github.com/virufy6/virufy6.github.io/assets/165954957/1adeae82-9314-4531-9996-e54ec24f3340">

- Removed extra lines of code related to the internationalization of the website using different languages.

- Changed the alt text for the homepage image in section 2 from Spanish to English. 

**Once the merge is approved it can be moved to QA**

a-wong-8 reviewed (commented):
looks like there is a new problem now after 1440px:
![Screenshot 2024-06-18 at 6 07 07 PM](https://github.com/virufy6/virufy6.github.io/assets/127047078/9cfb3967-7808-4230-ae00-c7419d214905)

ridham-dd committed:
fixed the overlapping text at 1440px,

ridham-virufy commented:
> looks like there is a new problem now after 1440px: ![Screenshot 2024-06-18 at 6 07 07 PM](https://private-user-images.githubusercontent.com/127047078/340879198-9cfb3967-7808-4230-ae00-c7419d214905.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg4MjYwMjksIm5iZiI6MTcxODgyNTcyOSwicGF0aCI6Ii8xMjcwNDcwNzgvMzQwODc5MTk4LTljZmIzOTY3LTc4MDgtNDIzMC1hZTAwLWM3NDE5ZDIxNDkwNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxOVQxOTM1MjlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MWU3Zjk4MTI4ZDY5MDA0ZmIxODRmODg1OThjYzA0NWY3NjU4NDQ2MmJlNDVlNjJlYmNlMGNmNGI5YTkzZTEzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.8m9q9umUcK5pRPXysSafi45Ut80IXqwmI4pCAT3KOAg)

It is fixed now, Should I move it to QA now?

gevuong commented:
So at 4k, the "Your Health, Our Priority" blue card stretches. If it's possible and you have time, I would set a max width for it. For example, if you look at `/our-story`, the card in the Values section does not stretch beyond a max width: https://deploy-preview-30--creative-jalebi-f8e18b.netlify.app/our-story

<img width="1157" alt="Screenshot 2024-06-19 at 1 46 13 PM" src="https://github.com/virufy6/virufy6.github.io/assets/27314227/0314022d-ae7f-4b67-bf7a-6809caf4412a">

It looks like the "How It Works" section stretches as well. If it's possible and you have time, I would set a max width for that section so that it doesn't stretch to the edge of the screen for larger screens (ie. 4k), and it will more so align with the Figma design and other sections (ie. hero, algorithms).
<img width="1154" alt="Screenshot 2024-06-19 at 1 46 00 PM" src="https://github.com/virufy6/virufy6.github.io/assets/27314227/50661bcb-c6fd-4260-9493-dcd5b414c0af">

Other than that, great job.

ridham-dd committed:
Fixed excess streching of "How It Works" section

ridham-virufy commented:
> So at 4k, the "Your Health, Our Priority" blue card stretches. If it's possible and you have time, I would set a max width for it. For example, if you look at `/our-story`, the card in the Values section does not stretch beyond a max width: https://deploy-preview-30--creative-jalebi-f8e18b.netlify.app/our-story
> 
> <img alt="Screenshot 2024-06-19 at 1 46 13 PM" width="1157" src="https://private-user-images.githubusercontent.com/27314227/341201217-0314022d-ae7f-4b67-bf7a-6809caf4412a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg4MzMyNTcsIm5iZiI6MTcxODgzMjk1NywicGF0aCI6Ii8yNzMxNDIyNy8zNDEyMDEyMTctMDMxNDAyMmQtYWU3Zi00YjY3LWJmN2EtNjgwOWNhZjQ0MTJhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE5VDIxMzU1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI2M2JhYmNjYjI1YzM5MjA1M2NhZmQ5NDQ5NzljZTA4YzEwYjhhOTdlYmE2NGE3ZGM1MmQ3YzRlMDM1MjBkODgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MidN8VSIdmNgkFJTzpQpAGXhCY-qI6ZEkakcaZEIkKM">
> It looks like the "How It Works" section stretches as well. If it's possible and you have time, I would set a max width for that section so that it doesn't stretch to the edge of the screen for larger screens (ie. 4k), and it will more so align with the Figma design and other sections (ie. hero, algorithms). <img alt="Screenshot 2024-06-19 at 1 46 00 PM" width="1154" src="https://private-user-images.githubusercontent.com/27314227/341201232-50661bcb-c6fd-4260-9493-dcd5b414c0af.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg4MzMyNTcsIm5iZiI6MTcxODgzMjk1NywicGF0aCI6Ii8yNzMxNDIyNy8zNDEyMDEyMzItNTA2NjFiY2ItYzZmZC00MjYwLTk0OTMtZGNkNWI0MTRjMGFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE5VDIxMzU1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgxYjJjZmU2MDJhYTA5Mzc0M2IwNjkzZDYyNDUxYWI3ODM1MzljOTYwNTZmODNlMTZjNWUzMWM1YTUxOGUzMjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.CH2Mt0jmat52-McoI4AanXcmj7sZNMYrshIk7ayY2CE">
> 
> Other than that, great job.

I have fixed the excess stretching of "How it works", and it should be smooth and good now. 

For "Your Health, Our Priority" card, making it not stretch was my implementation at first but later I changed it as it was not looking good to me on a personal level. If you observe the Our Values card in the deploy link you gave, it has a good background image for the emptiness and hence, it makes sense to not make it stretch. Here the background is black color and hence, it is not appealing enough. (Refer the image below)
![image](https://github.com/virufy6/virufy6.github.io/assets/165954957/1c984739-8c00-4354-82b7-8863e4cdbf20)

We can do two things now:
1) Ask UI team to update the design to have a background image for that section
or
2) decide if we want the blue card to stretch or not stretch with a black background.

gevuong commented:
I see. Sure lets just leave it as-is for now with the blue card stretching, and we can ask the UI team if they want the blue card to stretch or not during QA.

gevuong commented:
@a-wong-8 and @ridham-virufy, should the responsive webpages we build be supporting Mobile S - 320px as well?
Currently, I believe all the webpages Anthony and I have built support screens from 320px to 4k, so for consistency, I think this page should do so as well. What do you all think? I'm asking because I notice there is overlapping text at Mobile S - 320px.

Here's link to the Slack discussion regarding 320px. We have since fixed those issues mentioned in the thread: https://virufy.slack.com/archives/C076DU42PRQ/p1715888952371919

a-wong-8 commented:
@gevuong I say yes to 320px

ridham-dd committed:
made edits so that ti works with 320px viewport

ridham-virufy commented:
> @a-wong-8 and @ridham-virufy, should the responsive webpages we build be supporting Mobile S - 320px as well? Currently, I believe all the webpages Anthony and I have built support screens from 320px to 4k, so for consistency, I think this page should do so as well. What do you all think? I'm asking because I notice there is overlapping text at Mobile S - 320px.
> 
> Here's link to the Slack discussion regarding 320px. We have since fixed those issues mentioned in the thread: https://virufy.slack.com/archives/C076DU42PRQ/p1715888952371919

Made edits so that it works in 320px viewport! 

![Screenshot 2024-06-19 at 6 46 39 PM](https://github.com/virufy6/virufy6.github.io/assets/165954957/4043d3d2-0cdb-4cd0-bb87-f8054132565e)
![Screenshot 2024-06-19 at 6 49 33 PM](https://github.com/virufy6/virufy6.github.io/assets/165954957/bfd95ba5-860c-4a56-b020-bf03426742b1)
![image](https://github.com/virufy6/virufy6.github.io/assets/165954957/0acec72e-7f97-48f9-bf37-bb290ba14ed3)

ridham-dd committed:
After discussion with sherrie, middle section should be stretching was the outcome

ridham-virufy commented:
> > So at 4k, the "Your Health, Our Priority" blue card stretches. If it's possible and you have time, I would set a max width for it. For example, if you look at `/our-story`, the card in the Values section does not stretch beyond a max width: https://deploy-preview-30--creative-jalebi-f8e18b.netlify.app/our-story
> > <img alt="Screenshot 2024-06-19 at 1 46 13 PM" width="1157" src="https://private-user-images.githubusercontent.com/27314227/341201217-0314022d-ae7f-4b67-bf7a-6809caf4412a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg4MzMyNTcsIm5iZiI6MTcxODgzMjk1NywicGF0aCI6Ii8yNzMxNDIyNy8zNDEyMDEyMTctMDMxNDAyMmQtYWU3Zi00YjY3LWJmN2EtNjgwOWNhZjQ0MTJhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE5VDIxMzU1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI2M2JhYmNjYjI1YzM5MjA1M2NhZmQ5NDQ5NzljZTA4YzEwYjhhOTdlYmE2NGE3ZGM1MmQ3YzRlMDM1MjBkODgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MidN8VSIdmNgkFJTzpQpAGXhCY-qI6ZEkakcaZEIkKM">
> > It looks like the "How It Works" section stretches as well. If it's possible and you have time, I would set a max width for that section so that it doesn't stretch to the edge of the screen for larger screens (ie. 4k), and it will more so align with the Figma design and other sections (ie. hero, algorithms). <img alt="Screenshot 2024-06-19 at 1 46 00 PM" width="1154" src="https://private-user-images.githubusercontent.com/27314227/341201232-50661bcb-c6fd-4260-9493-dcd5b414c0af.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg4MzMyNTcsIm5iZiI6MTcxODgzMjk1NywicGF0aCI6Ii8yNzMxNDIyNy8zNDEyMDEyMzItNTA2NjFiY2ItYzZmZC00MjYwLTk0OTMtZGNkNWI0MTRjMGFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE5VDIxMzU1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgxYjJjZmU2MDJhYTA5Mzc0M2IwNjkzZDYyNDUxYWI3ODM1MzljOTYwNTZmODNlMTZjNWUzMWM1YTUxOGUzMjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.CH2Mt0jmat52-McoI4AanXcmj7sZNMYrshIk7ayY2CE">
> > Other than that, great job.
> 
> I have fixed the excess stretching of "How it works", and it should be smooth and good now.
> 
> For "Your Health, Our Priority" card, making it not stretch was my implementation at first but later I changed it as it was not looking good to me on a personal level. If you observe the Our Values card in the deploy link you gave, it has a good background image for the emptiness and hence, it makes sense to not make it stretch. Here the background is black color and hence, it is not appealing enough. (Refer the image below) ![image](https://private-user-images.githubusercontent.com/165954957/341209496-1c984739-8c00-4354-82b7-8863e4cdbf20.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg4NDQ0ODcsIm5iZiI6MTcxODg0NDE4NywicGF0aCI6Ii8xNjU5NTQ5NTcvMzQxMjA5NDk2LTFjOTg0NzM5LThjMDAtNDM1NC04MmI3LTg4NjNlNGNkYmYyMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyMFQwMDQzMDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hMjJhMWNiODBjMjM4NmQ2MzJmNDEyMWIxODVkOWI1MzYxYjkzZWMxOGJiODYzNDY2ODVkYWFmOTI3YzU1N2M4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.EJL37lJDHcobO60UiUYhhF1OBL1u3DMnNOqLNGiZdOI)
> 
> We can do two things now:
> 
> 1. Ask UI team to update the design to have a background image for that section
>    or
> 2. decide if we want the blue card to stretch or not stretch with a black background.

Sherrie suggested keeping the "Your Health, Our Priority" card to stretch, so the code is updated to do so.

gevuong commented:
Can you remove the `/home` route? Clicking `Home` on the navbar should redirect to `/`. It doesn't need to be appended with `/home` because the content at `/` is the same.

Update: Has been resolved in a separate merged PR.

gevuong commented:
>I do need help figuring out why the header and footer are not looking the desired way, that is having a transparent background. It might work once Homepage and the updated header by @a-wong-8 is merged together into the release.

We should try to figure this out before merging the branch. Can you elaborate on what you mean by the footer is not looking the desired way?

a-wong-8 commented:
@ridham-virufy it looks like you don't have the latest NAV bar and footer. Try running git pull origin <branch name>

ridham-virufy commented:
> > I do need help figuring out why the header and footer are not looking the desired way, that is having a transparent background. It might work once Homepage and the updated header by @a-wong-8 is merged together into the release.
> 
> We should try to figure this out before merging the branch. Can you elaborate on what you mean by the footer is not looking the desired way?

I pulled all the commits, but I don't think it helped in updation of the navbar and footer.
![image](https://github.com/virufy6/virufy6.github.io/assets/165954957/86e13aaa-1389-43d8-945e-4452507e56e5)

gevuong commented:
@ridham-virufy how are you pulling changes from release? Those changes you pulled are appearing as new changes in this PR and adding 20 commits..

After pulling, the commit should look more like this, as shown in this PR https://github.com/virufy6/virufy6.github.io/pull/36:
<img width="872" alt="Screenshot 2024-06-26 at 9 31 41 AM" src="https://github.com/virufy6/virufy6.github.io/assets/27314227/e012ef90-e3e2-4c85-ad64-8759e032f4ce">

Did you run `git pull origin release`?

_Edit: I'm also noticing this in your other PR as well. It's adding a lot of unnecessary code changes and 30 commits, making PR https://github.com/virufy6/virufy6.github.io/pull/34 more difficult to review and determine what were your code changes and what wasn't. If possible, I would either_

_1. revert how you git pulled in both of your branches_
_2. or create a new branch with only your changes, then run `git pull origin release`._

ridham-virufy commented:
> @ridham-virufy how are you pulling changes from release? Those changes you pulled are appearing as new changes in this PR and adding 20 commits..
> 
> After pulling, the commit should look more like this, as shown in this PR #36: <img alt="Screenshot 2024-06-26 at 9 31 41 AM" width="872" src="https://private-user-images.githubusercontent.com/27314227/343247176-e012ef90-e3e2-4c85-ad64-8759e032f4ce.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk0MjE2MDAsIm5iZiI6MTcxOTQyMTMwMCwicGF0aCI6Ii8yNzMxNDIyNy8zNDMyNDcxNzYtZTAxMmVmOTAtZTNlMi00Yzg1LWFkNjQtODc1OWUwMzJmNGNlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI2VDE3MDE0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEyNDY5ODE4MTI4MjIxOTU1ZTc5ZTA5NDE4M2RhMDgyOTFmMzg0NjdjMWNkMWM3MDBiOTgzNTRmZjY2ZmEyNjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.boyeLcEXheHIsT0xlAiJsV-FJCsRBV2-yVkYycoP8Zo">
> 
> Did you run `git pull origin release`?
> 
> _Edit: I'm also noticing this in your other PR as well. It's adding a lot of unnecessary code changes and commits, making PR #34 more difficult to review and determine what were your code changes and what wasn't. If possible, I would either_ _1. revert how you git pulled in both of your branches_ _2. or create a new branch with only your changes, then run `git pull origin release`._

Yes, the command I ran was `git pull origin release` I am not sure why all of those changes are reflected in the commit history. I can revert both the pull's but is there another way you want me to pull recent push from release? 
Do you think trying to merge the header footer branch into mine and then pushing it to release will fix the deploy preview's header and footer?

gevuong commented:
@ridham-virufy
1. Can you first revert the git pull so we can have your branch in the state it originally was?
2. Since you pulled from release, it should already contain the header and footer changes. If the issue still appears in preview, then that did not fix the transparent issue.
3. I asked this earlier but haven't received a response: Can you elaborate on what you mean by the footer is not looking the desired way?

ridham-dd committed:
one minor padding issue resolved

ridham-virufy commented:
> @ridham-virufy
> 
> 1. Can you first revert the git pull so we can have your branch in the state it originally was?
> 2. Since you pulled from release, it should already contain the header and footer changes. If the issue still appears in preview, then that did not fix the transparent issue.
> 3. I asked this earlier but haven't received a response: Can you elaborate on what you mean by the footer is not looking the desired way?

1. I have reverted all the extra commits.
2. I started working on Home page before Anthony was done with Header so, when I started the header was not completed. 
3. Footer is perfect, the only issues is transparency of the Header.

a-wong-8 commented:
@ridham-virufy this is how you fix the navbar issue in home.tsx:
```
    <div className="flex flex-col items-center justify-center **-mb-24**">
      <div className="**relative -top-24** w-full overflow-hidden">

```
<img width="1417" alt="Screenshot 2024-06-26 at 12 57 17 PM" src="https://github.com/virufy6/virufy6.github.io/assets/127047078/ddc5c367-b7be-46f4-a100-1fc4bf1a7e2c">

gevuong commented:
@ridham-virufy Thank you for reverting the commits. It helps me debug the issue. Ok I think what's going on is the branches you create are not branched off of `release`, which may explain why every time you pull from `release`, those changes you pull are considered new changes..

When you create a new branch, it needs to branch off of `release`. One way to do that is to first make sure you're on the `release` branch, run `git checkout -b <insert new branch name>`. That newly created branch should then be a branch of `release`.

_Edit: From there, I would add your page changes to that branch. I would do the same with the other PR._

ridham-virufy commented:
> @ridham-virufy Thank you for reverting the commits. It helps me debug the issue. Ok I think what's going on is the branches you create are not branched off of `release`, which may explain why every time you pull from `release`, those changes you pull are considered new changes..
> 
> When you create a new branch, it needs to branch off of `release`. One way to do that is to first make sure you're on the `release` branch, run `git checkout -b <insert new branch name>`. That newly created branch should then be a branch of `release`.
> 
> _Edit: From there, I would add your page changes to that branch. I would do the same with the other PR._



> @ridham-virufy Thank you for reverting the commits. It helps me debug the issue. Ok I think what's going on is the branches you create are not branched off of `release`, which may explain why every time you pull from `release`, those changes you pull are considered new changes..
> 
> When you create a new branch, it needs to branch off of `release`. One way to do that is to first make sure you're on the `release` branch, run `git checkout -b <insert new branch name>`. That newly created branch should then be a branch of `release`.
> 
> _Edit: From there, I would add your page changes to that branch. I would do the same with the other PR._

I have been creating branches manually as shown in the screenshot below. I am pretty sure it is branched off from release. 
<img width="1512" alt="Screenshot 2024-06-26 at 3 43 47 PM" src="https://github.com/virufy6/virufy6.github.io/assets/165954957/ed4e8cec-86ac-4a7c-9ea6-ac7171c0d77a">

Let me know what you want me to do.

gevuong committed:
Merge branch 'release' of https://github.com/virufy6/virufy6.github.io into Homepage-ridham

gevuong commented:
Ok here, I just ran `git pull origin release` on your branch and it worked..
<img width="671" alt="Screenshot 2024-06-26 at 3 08 48 PM" src="https://github.com/virufy6/virufy6.github.io/assets/27314227/9145b1db-ea1f-4657-bf0f-3e9bf6f9729c">

I think it may have to do with the merge strategy you have set up. To confirm, can you share screenshot of the output when you run `git config --list`?

ridham-virufy commented:
> git config --list

![image](https://github.com/virufy6/virufy6.github.io/assets/165954957/5ad05173-fb37-411b-92c1-616ea15ab1a2)
Does it look good?

gevuong commented:
Ok I figured it out. You are rebasing on a public branch (ie. `release`) which goes agains the rule of rebasing. What that will do is all the commits from `release` will be added on top of your feature branch. If you're not familiar with rebasing and when to use it, I recommend setting your pull rebase config to false. To do this, run `git config pull.rebase false`.

Now, whenever you pull from a public branch (ie. `release`), all the commits from `release` will not be added on top of your feature branch. Instead, it will merge and occupy a single commit.

ridham-dd committed:
QA comment closed (Rupak's)

ridham-dd committed:
Updated the code as suggested by Anthony (Doesn't work though)

ridham-dd committed:
Header fixed!! Yayyyy

ridham-dd committed:
margin-top fixed

ridham-virufy committed:
Merge branch 'release' into Homepage-ridham
